### PR TITLE
Task_DEV-1

### PR DESCRIPTION
**Pull request by MarinaGotka**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+namespace task1
+{
+    class Program
+    {
```
Comment (IgorKheidorov) [inline Program.cs]: Read task very carefully, do not satisfy task conditions

File: Program.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+namespace task1
+{
+    class Program
```
Comment (IgorKheidorov) [inline Program.cs]: Bad name, name of class has to be meaningfull

Comment (IgorKheidorov) [inline Program.cs]: Bad name, name of class has to be meaningful

File: Program.cs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    {
+        static void Main(string[] args)
+        {
+            int max_count = 0;
```
Comment (IgorKheidorov) [inline Program.cs]: C-style name? not C#. Code style violation

File: Program.cs [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+                    current_count = 1;
+                }
+            }
+
+            if (current_count > max_count)
+            {
+                max_count = current_count;
+            }
+            Console.WriteLine(max_count);
+            Console.Read();
```
Comment (IgorKheidorov) [inline Program.cs]: Read what? Why?

File: Program.cs [hunk 5]
```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+namespace task1
+{
```
Comment (IgorKheidorov) [inline Program.cs]: No comments to class and method

File: Program.cs [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+                    }
+                }
+
+                if (currentCount > maxCount)
+                {
+                    maxCount = currentCount;
+                }
+                Console.WriteLine("Maximum = {0}", maxCount);
+            }
+            else Console.WriteLine("There is no parametres");
```
Comment (IgorKheidorov) [inline Program.cs]: code style

File: Program.cs [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+namespace task_DEV1
+{
+    class Entry  //Class for finding the maximum number of identical symbols
+    {
+        static void Main(string[] args) //finding the maximum number of identical symbols
+        {
+            int maxCount = 0;
+            int currentCount = 1;
+
+            if(args.Length>1)
```
Comment (IgorKheidorov) [inline Program.cs]: What about 100 parameters? its not normal

File: Program.cs [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+            {
+                string str = args[0];
```
Comment (IgorKheidorov) [inline Program.cs]: Why not use args[0] directly

File: Program.cs [hunk 9]
```diff
@@ -0,0 +1,41 @@
+﻿using System;
+
+namespace task_DEV1
+{
+    class Entry  //Class for finding the maximum number of identical symbols
+    {
+        static void Main(string[] args) //finding the maximum number of identical symbols
```
Comment (IgorKheidorov) [inline Program.cs]: comments to method

File: Program.cs [hunk 10]
```diff
@@ -0,0 +1,41 @@
+﻿using System;
+
+namespace task_DEV1
+{
```
Comment (IgorKheidorov) [inline Program.cs]: comments to class

File: Program.cs [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+    class Entry  //Class for finding the maximum number of identical symbols
```
Comment (AlexanderSoroka) [inline Program.cs]: Wrong comment format - please, use documentation comment https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/xml-documentation-comments

File: Program.cs [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+    class Entry
+    {
+        /// <summary>
+        /// Method Main() 
+        /// is the entry point
+        /// </summary>
+        /// <param name="args">Method argument of Main()</param>
+        static void Main(string[] args)
+        {
+            if (args.Length >= 1)
```
Comment (IgorKheidorov) [inline Program.cs]: Bad logics, args.Length has to be = 1, other cases is input error

File: Program.cs [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+    ///  To find and print the maximum number of repeated 
+    ///  consecutive elements in a row
+    /// </summary>
+    class Entry
+    {
+        /// <summary>
+        /// Method Main() 
+        /// is the entry point
+        /// </summary>
+        /// <param name="args">Method argument of Main()</param>
```
Comment (IgorKheidorov) [inline Program.cs]: Bad comment

File: Program.cs [hunk 14]
```diff
@@ -0,0 +1,58 @@
...
+        static void Main(string[] args)
+        {
+            if (args.Length >= 1)
+            {
+                for (int i = 0; i < args.Length; i++)
```
Comment (IgorKheidorov) [inline Program.cs]: error

File: Program.cs [hunk 15]
```diff
@@ -0,0 +1,56 @@
+﻿using System;
+
+namespace task_DEV1
+{
+    /// <summary>
+    ///  Class Entry
+    ///  Main Class of program
```
Comment (AlexanderSoroka) [inline Program.cs]: Line length can be 120 symbols, please do not break line to multiple lines without real reason

File: Program.cs [hunk 16]
```diff
@@ -0,0 +1,56 @@
...
+    /// </summary>
+    class Entry
+    {
+        /// <summary>
+        /// Method Main() 
+        /// is the entry point
+        /// It find and print the maximum number of 
+        /// identical consecutive elements in a row
+        /// </summary>
+        /// <param name="args"> Method argument of Main()
```
Comment (AlexanderSoroka) [inline Program.cs]: invalid param description

File: Program.cs [hunk 17]
```diff
@@ -0,0 +1,56 @@
...
+        /// It find and print the maximum number of 
+        /// identical consecutive elements in a row
+        /// </summary>
+        /// <param name="args"> Method argument of Main() 
+        /// is string for processing </param>
+        static void Main(string[] args)
+        {
+            if (args.Length == 1)
+            {
+                    int maxCount = 0;
```
Comment (AlexanderSoroka) [inline Program.cs]: move it to separate method

File: Properties/AssemblyInfo.cs [hunk 18]
```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// Общие сведения об этой сборке предоставляются следующим набором
```
Comment (AlexanderSoroka) [inline Properties/AssemblyInfo.cs]: Do not use Russian

**Comment by MarinaGotka** [comment]
I have deleted 1.txt

**Pull request merged**